modules/profile/public/lib/commands/profiles/create/build.liquid

```diff
@@ -2,10 +2,8 @@
   function tokenized_names = 'modules/profile/commands/profiles/tokenize_names', object: object
   assign uuid_new = '' | uuid
   assign uuid = object.uuid | default: uuid_new
-  assign name = object.first_name | append: ' ' | append: object.last_name
```

Comment:
we still use `name` in line number 6 - so we should remove it form there; are we sure we don't need `name` though?